clayui.com/content/docs/components/markup-treeview.md

```diff
@@ -0,0 +1,7751 @@
+---
+title: 'Treeview'
+description: 'A tree view is a component-based on nodes that are shown in a hierarchical structure.'
+lexiconDefinition: 'https://liferay.design/lexicon/core-components/tree-view/'
+---
+
+<div class="nav-toc-absolute">
+<div class="nav-toc">
+
+-   [Treeview](#css-treeview)
+    -   [Group](#css-treeview-group)
+    -   [Item](#css-treeview-item)
+    -   [Link](#css-treeview-link)
+    -   [Component Expander](#css-treeview-component-expander)
+    -   [Component Action](#css-treeview-component-action)
+-   [Variations](#css-variations)
+    -   [Expander on Hover](#css-treeview-expander-on-hover)
+    -   [Light](#css-treeview-light)
+    -   [Dark](#css-treeview-dark)
+
+</div>
+</div>
+
+<div class="clay-site-alert alert alert-warning">This component is a Cadmin only component.</div>
+
+The component must be wrapped in:
+
+```html{expanded}
+<div class="cadmin">
+	...
+</div>
+```
+
+## Treeview(#css-treeview)
+
+The treeview provides a way to display information in a hierarchical structure by using collapsible items (nodes). You can navigate between these items using either your mouse device or keyboard. The most common example of a treeview is a folder structure for file systems, but it can be used for showing any hierarchical relationships.
+
+<div class="clay-site-alert alert alert-info">
+	See <a href="https://www.w3.org/TR/wai-aria-practices-1.1/examples/treeview/treeview-2/treeview-2a.html" rel="noreferrer noopener" target="_blank">https://www.w3.org/TR/wai-aria-practices-1.1/examples/treeview/treeview-2/treeview-2a.html</a> for accessibility patterns.
+</div>
+
+<div class="clay-site-alert alert alert-warning">
+	Treeview Link indentation must be provided by javascript through inline styles, use the styles `padding-left: 24px;` on `.treeview-link` and `margin-left: -24px` on `.treeview-link > .c-inner`. Increase the indentation for each level by increasing or decreasing `padding-left` or `margin-left` by 24px.
+</div>
+
+<div class="cadmin">
+	<ul class="treeview treeview-light treeview-nested" role="tree">
```

Comment:
I saw that you changed the name of the class `treeview-nested` to `treeview-nested-margins`, should we replace it here too?

Reply:
@matuzalemsteles `treeview-nested-margins` is the type with the dead space on the left. I left it in as an alternative. `treeview-nested` is the correct name for the version that uses padding. It gives the option to set padding via CSS if for whatever reason a dev can't use the React plugin.

Reply:
Makes sense. Let's merge this. Do you have anything else to work on in the Treeview?

Reply:
Nothing else, I'm going to merge this.